internal/services/storage/migration/account.go

```diff
@@ -204,3 +211,810 @@ func accountSchemaForV0AndV1() map[string]*pluginsdk.Schema {
+						Type:     pluginsdk.TypeList,
+						Optional: true,
+						MaxItems: 1,
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*pluginsdk.Schema{
+								"days": {
+									Type:         pluginsdk.TypeInt,
+									Optional:     true,
+									Default:      7,
+									ValidateFunc: validation.IntBetween(1, 365),
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+
+		"queue_properties": {
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			Computed: true,
+			MaxItems: 1,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"cors_rule": helpers.SchemaStorageAccountCorsRule(false),
+					"logging": {
+						Type:     pluginsdk.TypeList,
+						Optional: true,
+						MaxItems: 1,
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*pluginsdk.Schema{
+								"version": {
+									Type:         pluginsdk.TypeString,
+									Required:     true,
+									ValidateFunc: validation.StringIsNotEmpty,
+								},
+								"delete": {
+									Type:     pluginsdk.TypeBool,
+									Required: true,
+								},
+								"read": {
+									Type:     pluginsdk.TypeBool,
+									Required: true,
+								},
+								"write": {
+									Type:     pluginsdk.TypeBool,
+									Required: true,
+								},
+								"retention_policy_days": {
+									Type:         pluginsdk.TypeInt,
+									Optional:     true,
+									ValidateFunc: validation.IntBetween(1, 365),
+								},
+							},
+						},
+					},
+					"hour_metrics": {
+						Type:     pluginsdk.TypeList,
+						Optional: true,
+						MaxItems: 1,
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*pluginsdk.Schema{
+								"version": {
+									Type:         pluginsdk.TypeString,
+									Required:     true,
+									ValidateFunc: validation.StringIsNotEmpty,
+								},
+								"enabled": {
+									Type:     pluginsdk.TypeBool,
+									Required: true,
+								},
+								"include_apis": {
+									Type:     pluginsdk.TypeBool,
+									Optional: true,
+								},
+								"retention_policy_days": {
+									Type:         pluginsdk.TypeInt,
+									Optional:     true,
+									ValidateFunc: validation.IntBetween(1, 365),
+								},
+							},
+						},
+					},
+					"minute_metrics": {
+						Type:     pluginsdk.TypeList,
+						Optional: true,
+						MaxItems: 1,
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*pluginsdk.Schema{
+								"version": {
+									Type:         pluginsdk.TypeString,
+									Required:     true,
+									ValidateFunc: validation.StringIsNotEmpty,
+								},
+								"enabled": {
+									Type:     pluginsdk.TypeBool,
+									Required: true,
+								},
+								"include_apis": {
+									Type:     pluginsdk.TypeBool,
+									Optional: true,
+								},
+								"retention_policy_days": {
+									Type:         pluginsdk.TypeInt,
+									Optional:     true,
+									ValidateFunc: validation.IntBetween(1, 365),
```

Comment:
fwiw you don't need these for a state migration - only the Type and Required/Optional/Computed iirc

```suggestion
```